src/remote.go

```diff
@@ -551,13 +551,8 @@ func (r *Remote) Download(id string, exportURL string) (io.ReadCloser, error) {
 	var url string
 	var body io.ReadCloser
 
-	if len(exportURL) < 1 {
-		url = DriveResourceHostURL + id
-	} else {
-		url = exportURL
-	}
+	resp, err := r.service.Files.Get(id).Download()
```

Comment:
While at lunch, I just noticed that here we've gotten rid of exporting files. We should update it to allow exporting of files.

An updated fix will be something like this
```go
var res *http.Response
var err error

if len(exportURL) < 1 {
    res, err = r.service.Files.Get(id).Download()
} else {
    res, err = r.client.Get(exportURL)
}
```

My apologies, in the morning I was in a rush to get to work so didn't have to properly review it.